geomstats/_backend/tensorflow/__init__.py

```diff
@@ -787,3 +791,47 @@ def tile(x, multiples):
     t2 = tf.shape(x)
     x_reshape = tf.reshape(x, tf.concat([t1, t2], axis=0))
     return tf.tile(x_reshape, multiples)
+
+
+def vec_to_triu(vec):
+    """Takes vec (batch of vecs) and forms
```

Comment:
NIT: needs to respect our docstring conventions (Takes -> Take, only one line for the short description, the rest of the docstring goes in the longer description).

Reply:
Done

> NIT: needs to respect our docstring conventions (Takes -> Take, only one line for the short description, the rest of the docstring goes in the longer description).

Done